package.json

```diff
@@ -24,6 +24,7 @@
     "publish:master": "lerna publish --canary --force-publish --preid=alpha --exact --include-merged-tags --conventional-prerelease --yes",
     "publish:beta": "lerna publish --exact --dist-tag=beta --preid=beta --conventional-commits --conventional-prerelease --message 'chore(release): Publish [ci skip]' --yes",
     "publish:release": "lerna publish --conventional-commits --exact --yes --message 'chore(release): Publish [ci skip]'",
+    "publish:headless-staging": "lerna publish --canary --force-publish --preid=headless --dist-tag=headless --exact --include-merged-tags --conventional-prerelease --yes",
```

Comment:
Do we need this in master?

Reply:
we don't need this in master but I'd like to leave it for now because I think it will create merge conflicts whenever I merge master back into the headless branch if we remove it